lib/crypto.js

```diff
@@ -2,23 +2,29 @@
 
 var crypto = require('crypto');
 
-function encrypt(key, text){
-  var IV = new Buffer(crypto.randomBytes(16));
-  var cipher = crypto.createCipheriv('aes-256-ctr', key, IV)
-  var crypted = cipher.update(text,'utf8','hex')
+function encrypt(key, text) {
+  const hashedAccessKey = crypto.createHash('sha256').update(key).digest('hex');
```

Comment:
I didn't see `const` being used throughout the project, so I think its better to not use it, assuming that they're trying to keep it with old JS stuff